include/rocksdb/options.h

```diff
@@ -1010,6 +1010,8 @@ struct ReadOptions {
   // Default: nullptr
   const Slice* iterate_upper_bound;
 
+  const Slice* stop_key;
```

Comment:
what's the difference between this option and `iterate_upper_bound`?

Also btw I thought we were going to limit this PR's scope to the mutable CF option, and put the next changes (deciding which bloom filters can be used based on upper-bound) in a later PR. Is it still possible? I am worried this one grows too long for people to understand.

Reply:
Sure, I can revert the last commit since it's still experimental